app/services/mixtape_maker.rb

```diff
@@ -0,0 +1,15 @@
+class MixtapeMaker
+
+  def self.generate(no_song = 5, artists:)
```

Comment:
For named arguments, you can use:

`generate(no_song: 5, artists:)`

no_song will default to 5 (though, I read this as "no song" ie (there is no song) and not say "number_of_songs")